src/test/resources/json/addFileStatus_data_all.json

```diff
@@ -0,0 +1,9 @@
+{
+  "data": {
+    "addFileStatus": {
+      "fileId": "07a3a4bd-0281-4a6d-a4c1-8fa3239e1313",
+      "statusType": "Upload",
+      "statusValue": "Success"
+    }
+  }
+}
```

Comment:
Can you add a line break at the end of the file please.

Reply:
Added, thanks.